docs/overview/persistence.md

```diff
@@ -0,0 +1,112 @@
+---
+id: overview_persistence
+title: "Persistence"
+---
+
+Persistence gives you ability to store events that occur in your system with defined datastore. 
+
+To use Persistence you need in your `build.sbt`:
+
+```scala mdoc:passthrough
+
+println(s"""```""")
+if (zio.actors.BuildInfo.isSnapshot)
+  println(s"""resolvers += Resolver.sonatypeRepo("snapshots")""")
+println(s"""libraryDependencies += "dev.zio" %% "zio-actors-persistence" % "${zio.actors.BuildInfo.version}"""")
+println(s"""```""")
+
+```
+
+For current version the only datastore available is `postgresql` and in-memory datastore for testing purposes. 
+For `postgresql` you need a configuration in (by default) `resources/application.conf`:
+
+```hocon
+ActorSystemName.zio.actors.persistence {
+  plugin = "jdbc-journal"
+  url = "jdbc:postgresql://localhost:5432/postgres"
+  user = "user"
+  pass = "pass"
+}
+```
+
+Currently the table that needs to be present in database has such schema:
```

Comment:
Who should perform this migration? If it's up to us, and I think it should be, then don't mention it here.

Reply:
Sure, so I will add performing migration in initialization of jdbc plugin. Will remove it.